bin/faucet/README.md

```diff
@@ -46,5 +46,138 @@ The faucet implements several security measures to prevent abuse:
   - Requests are processed in batches to optimize performance
   - Failed requests within a batch are handled individually
 
+## Usage
+
+The faucet can be accessed by the HTTP API interactively through the frontend or programmatically by building the requests manually.
+
+### Programmatic API Usage
+
+The faucet provides a REST API. The typical flow to request tokens involves:
+
+1. **Request a Proof-of-Work challenge** from `/pow`
+
+```rust
+let base_url = "http://localhost:8080";
+let url = format!("{base_url}/pow?account_id=mlcl1qq8mcy8pdvl0cgqfkjzf8efjjsnlzf7q");
+let response = reqwest::get(&url).await?.error_for_status()?.text().await?;
+let json: serde_json::Value = serde_json::from_str(&response)?;
+let challenge: &str = json["challenge"].as_str().unwrap();
+let target: u64 = json["target"].as_u64().unwrap();
+```
```

Comment:
Is it possible to use TypeScript or JavaScript examples? My thinking for this is that the target user here is probably a web developer, and I'm assuming we already have such examples because we do this in the faucet front-end.

We can still have a separate Rust example file (i.e., the `bin/faucet/examples/request_tokens.rs`) but here it may make sense to show to to do these from the web.